changelogs/fragments/1780-zos_mvs_raw-verbose-fail.yml

```diff
@@ -0,0 +1,8 @@
+bugfixes:
+  - zos_mvs_raw - If a program failed with a non-zero return code and verbose was false, the module would succeed.
+    Whereas, if the program failed and verbose was true the module would fail.
```

Comment:
Does this sentence add anything? The module is supposed to fail when rc is nonzero, so this isn't describing a bug.
```suggestion
```